mypy/test/testtypes.py

```diff
@@ -1541,7 +1541,7 @@ def make_call(*items: tuple[str, str | None]) -> CallExpr:
 class TestExpandTypeLimitGetProperType(TestCase):
     # WARNING: do not increase this number unless absolutely necessary,
     # and you understand what you are doing.
-    ALLOWED_GET_PROPER_TYPES = 8
+    ALLOWED_GET_PROPER_TYPES = 9
```

Comment:
@JelleZijlstra The test here just count all `get_proper_type` calls inside `expandtype`. I believe the change still makes sense and is necessary to be able to compare the `type` inside `Unpack`. Furthermore, it's guarded by the `isinstance` check so the performance penalty should probably be minimal. Let me know what you think.

_No other test case so far would reach that, only the new TypeVarTuple default ones._